tests/plugins/image2/manual/aspectratio.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 4.9.0, bug, 1348
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, sourcearea, toolbar, image2, link, undo
+
+1. Double click on the image to popup dialog.
+2. Change url of the image to `/tests//_assets/lena.jpg`.
+3. Ensure that the image is locked.
+4. Change `width` to `400`.
+5. Click `OK` button.
+
+## Expected
+
+The image has preserved aspect ratio.
```

Comment:
I would go with `The inserted image has...` (so it is clear that image should be inserted). Same in **Unexpected**.